diff --git a/src/standards/cpm_1.rs b/src/standards/cpm_1.rs
index 632e31f..a734ec8 100644
--- a/src/standards/cpm_1.rs
+++ b/src/standards/cpm_1.rs
@@ -12,7 +12,7 @@ pub mod cpm_pdu_descriptions {
 
     use rasn::prelude::*;
 
-    use super::cam_pdu_descriptions::GenerationDeltaTime;
+    use super::super::cam_1_4_1::cam_pdu_descriptions::GenerationDeltaTime;
     use super::super::dsrc_2_2_1::etsi_its_dsrc::{
         IntersectionReferenceID,
         LaneID,
@@ -607,7 +607,7 @@ pub mod cpm_pdu_descriptions {
         }
     }
     fn originating_vehicle_container_drive_direction_default() -> DriveDirection {
-        FORWARD
+        DriveDirection::forward
     }
     #[derive(AsnType, Debug, Clone, Decode, Encode, PartialEq, Eq, Hash)]
     #[rasn(automatic_tags)]
