diff --git a/src/standards/dsrc_2_2_1.rs b/src/standards/dsrc_2_2_1.rs
index f4d9f1c..6578f64 100644
--- a/src/standards/dsrc_2_2_1.rs
+++ b/src/standards/dsrc_2_2_1.rs
@@ -13,35 +13,6 @@ pub mod etsi_its_dsrc {
     use rasn::prelude::*;
 
     use super::super::cdd_2_2_1::etsi_its_cdd::{Iso3833VehicleType, Latitude, Longitude, StationID};
-    use super::super::dsrc_2_2_1::etsi_its_dsrc_region::{
-        RegAdvisorySpeed,
-        RegComputedLane,
-        RegConnectionManeuverAssist,
-        RegGenericLane,
-        RegIntersectionGeometry,
-        RegIntersectionState,
-        RegLaneAttributes,
-        RegLaneDataAttribute,
-        RegMapData,
-        RegMovementEvent,
-        RegMovementState,
-        RegNodeAttributeSetXY,
-        RegNodeOffsetPointXY,
-        RegPosition3D,
-        RegRTCMcorrections,
-        RegRequestorDescription,
-        RegRequestorType,
-        RegRestrictionUserType,
-        RegRoadSegment,
-        RegSPAT,
-        RegSignalControlZone,
-        RegSignalRequest,
-        RegSignalRequestMessage,
-        RegSignalRequestPackage,
-        RegSignalStatus,
-        RegSignalStatusMessage,
-        RegSignalStatusPackage,
-    };
     #[doc = " Anonymous SEQUENCE OF member "]
     #[derive(AsnType, Debug, Clone, Decode, Encode, PartialEq, Eq, Hash)]
     #[rasn(automatic_tags, identifier = "SEQUENCE")]
