diff --git a/src/standards/ivim_2_2_1.rs b/src/standards/ivim_2_2_1.rs
index 302eb6e..3306cfd 100644
--- a/src/standards/ivim_2_2_1.rs
+++ b/src/standards/ivim_2_2_1.rs
@@ -822,7 +822,6 @@ pub mod ivi {
         InternationalSignRateOfIncline,
         InternationalSignSpeedLimits,
     };
-    use super::ivi_is::InfrastructureSupportSet;
     #[doc = " Extension in V3.1"]
     #[doc = "  Definition of Data Frames"]
     #[derive(AsnType, Debug, Clone, Decode, Encode, PartialEq, Eq, Hash)]
