diff --git a/autogen/asn.1/ivim_2_1_1-part2.asn b/autogen/asn.1/ivim_2_1_1-part2.asn
index e5e231a..4b1eb08 100644
--- a/autogen/asn.1/ivim_2_1_1-part2.asn
+++ b/autogen/asn.1/ivim_2_1_1-part2.asn
@@ -14,10 +14,10 @@ EuVehicleCategoryCode, Iso3833VehicleType
 FROM ElectronicRegistrationIdentificationVehicleDataModule {iso(1) standard(0) iso24534 (24534) vehicleData (1) version1 (1)}
 
 VarLengthNumber
-FROM CITSdataDictionary1 {iso(1) standard(0) cits-applMgmt (17419) dataDictionary (1) version1 (1)} 
+FROM CITSapplMgmtIDs {iso(1) standard(0) cits-applMgmt (17419) ids (1) version1 (1)}
 
 RoadSegmentReferenceID, IntersectionReferenceID, LaneID 
-FROM DSRC {iso (1) standard (0) signalizedIntersection (19091) profilec(2) dsrc (2) version2 (2)}
+FROM ETSI-ITS-DSRC {itu-t (0) identified-organization (4) etsi (0) itsDomain (5) wg1 (1) ts103301 (103301) dsrc (6) major-version-2 (2) minor-version-1 (1)}
 
 InternationalSign-applicablePeriod, InternationalSign-exemptedApplicablePeriod, InternationalSign-directionalFlowOfLane, InternationalSign-applicableVehicleDimensions, InternationalSign-speedLimits, InternationalSign-rateOfIncline, InternationalSign-distanceBetweenVehicles, InternationalSign-destinationInformation
 FROM GDD {iso (1) standard (0) gdd(14823) version1 (0)};
